distribution/pom.xml

```diff
@@ -28,6 +28,13 @@
   <name>Luna: Distribution</name>
   <description>Luna distribution that builds the Docker Image for the projects running on WebSight.</description>
 
+  <properties>
+    <websight.base.version>1.3.0</websight.base.version>
```

Comment:
I think using websight.base.version here might be hard for users. WebSight CMS also contains some version of  websight base. After bumb of CMS version this test might start to fail and this might be hard to understand why is it failing and why it.extra.bundles.count must be changed. It might not be clear for users when to bump this version and what should be value. 
I think that CMS should deliver tests module too (so test module version would be chaged by CMS version change) and users should specify in it.extra.bundles.count just the number of bundles added by them.
Also would be good if users would not have to add dependencies used by the tests.